public/js/main.js

```diff
@@ -187,10 +188,12 @@ function validateTemplateFieldsParams(query) {
  * @returns {boolean|void}
  */
 function validateGlobalQueryParams(params) {
-    if (params.queryId === undefined) {
-        return handleError({body: {message: 'Missing URL parameter “query_id”'}});
-    } else if (!Number.isInteger(+params.queryId)) {
-        return handleError({body: {message: 'URL parameter “query_id” must be a number'}});
+    if (params.queryId === undefined && params.queryName === undefined) {
+        return handleError({body: {message: 'Missing URL parameter: a “queryId” (number) or a “queryName” (string) is mandatory'}});
+    } else if (params.queryId !== undefined && params.queryName !== undefined) {
+        return handleError({body: {message: 'Only one query parameter is allowed: impossible to use “queryId” and “queryName” at the same moment'}});
+    } else if (params.queryId !== undefined && !Number.isInteger(+params.queryId)) {
+        return handleError({body: {message: 'URL parameter “queryId” must be a number'}});
     } else if (params.sourceKey === undefined) {
         return handleError({body: {message: 'Missing URL parameter “source_key” (must be a string)'}});
```

Comment:
@andre91a even if not related to the change, can you fix `source_key` in `sourceKey` since this is the good name?

Reply:
@fantandrea done